types/src/unit_tests/trusted_state_test.rs

```diff
@@ -490,9 +462,12 @@ proptest! {
         );
 
         let change_proof = ValidatorChangeProof::new(lis_with_sigs, false /* more */);
-        trusted_state
-            .verify_and_ratchet(&latest_li, &change_proof)
-            .expect_err("Should always return Err with a invalid latest li sigs");
+        // We don't verify ledger info <= known version
```

Comment:
Once it's properly done we should remove this `if`, right? This is a "// FIXME"

Reply:
No, even it's fixed, this'll not return error unless the ledger info content is different

Reply:
if the version equals, it should be a ::NoChange

Reply:
I think the test is about a invalid latest_li here?